### PR TITLE
[leone] Yade 2017 01a foss 2015 b

### PR DESCRIPTION
**Pull request by ChristopherBignamini**

### CODE REVIEW

File: easybuild/easyconfigs/m/minieigen/minieigen-0.5.3-Python-2.7.10.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'PythonPackage'
```
Comment (gppezzi) [inline easybuild/easyconfigs/m/minieigen/minieigen-0.5.3-Python-2.7.10.eb]: The name of the file is incomplete (toolchain and toolchainversion are missing).

File: easybuild/easyconfigs/h/h-yade/h-yade-2017.01a-foss-2015b.eb [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+configopts += '-DGL2PS_INCLUDE_DIR=$EBROOTGL2P/include '
+configopts += '-DEIGEN3_INCLUDE_DIR=$EBROOTEIGEN/include/ '
+configopts += '-DPYTHON_EXECUTABLE=$EBROOTPYTHON/bin/python '
+configopts += '-DPYTHON_INCLUDE_DIR=$EBROOTPYTHON/include/python2.7 '
+configopts += '-DPYTHON_LIBRARY=$EBROOTPYTHON/lib '
+configopts += '-DQGLVIEWER_INCLUDE_DIR=$EBROOTLIBQGLVIEWER/include '
+configopts += '-DUSE_QT5=OFF '
+configopts += '-DCMAKE_CXX_FLAGS="-DYADE_CPM_FULL_MODEL_AVAILABLE" '
+
+modextravars = {'LD_PRELOAD':'/apps/leone/UES/RH6.7_PE15.12/HILTI/EBSandBox/easybuild/software/GL2PS/1.3.8-foss-2015b/lib/libgl2ps.so'}
```
Comment (gppezzi) [inline easybuild/easyconfigs/h/h-yade/h-yade-2017.01a-foss-2015b.eb]: please try to comment out this line to see if this workaround is still needed in the upgraded OS.

If it does not work, be sure to update to the path to the latest installation (or we should find a way to expand the `$EBROOTGL2PS` variable here).

**Comment by lucamar** [comment]
retest this please

**Comment by gppezzi** [comment]
closing as it the [Yade version with python 2.7.12](https://github.com/eth-cscs/production/pull/473) replaces this PR.

**Pull request closed without merging**